packages/contracts/README.md

```diff
@@ -84,5 +84,39 @@ You can also build specific components as follows:
 yarn build:contracts
 ```
 
+### Deploying the Contracts
+To deploy the contracts first clone, install, and build the contracts package.
+
+Next set the following env vars:
+
+```bash
+CONTRACTS_TARGET_NETWORK=...
+CONTRACTS_DEPLOYER_KEY=...
+CONTRACTS_RPC_URL=...
+```
+
+Then to perform the actual deployment run:
+
+```bash
```

Comment:
I think it would be worth either:
1. Linking to the deploy task that has all of these config options or
2. Showing usage of `npx hardhat deploy --help` to view all available options

Reply:
For `2`, that can change and get out of sync but I do agree that is nice

Reply:
Went with displaying the result of `npx hardhat deploy --help`